tests/realtikvtest/scripts/bootstrap-common.sh

```
@@ -0,0 +1,62 @@
+        fi
+        sleep 1
+    done
+    echo "TiKV cluster did not become ready" >&2
+    tail -n 100 tikv0.log tikv1.log tikv2.log >&2 || true
+    return 1
+}
+
+function start_tikv_cluster() {
+    for attempt in {1..3}; do
+        echo "Starting TiKV cluster (attempt ${attempt}/3)"
+        bin/tikv-server --config=${config_dir}/tikv.toml --data-dir=${data_base_dir}tikv-0/data --addr=127.0.0.1:20160 --advertise-addr=127.0.0.1:20160 --status-addr=127.0.0.1:20180 --pd=${pd_client_urls} --log-file=tikv0.log &
```

Shellcheck (0.11.0)

[warning] config_dir is referenced but not assigned.

(SC2154)